src/groupeddataframe/grouping.jl

```diff
@@ -65,6 +65,14 @@ See the following for additional split-apply-combine operations:
 * [`map`](@ref) : apply a function to each group of a `GroupedDataFrame` (without combining)
 * [`combine`](@ref) : combine a `GroupedDataFrame`, optionally applying a function to each group
 
+`GroupedDataFrame` also supports the dictionary interface. The keys
+are the `GroupKey` objects returned by [`Base.keys(::GroupedDataFrame)`](@ref),
+which can also be used to get the values of the grouping columns for each group.
+`Tuples` and `NamedTuple`s containing the values of the grouping columns are
```

Comment:
I'd say "in the same order as the `cols` argument" here, and drop the last sentence.